### PR TITLE
fix: Content for Error fetching event (#1366)

### LINKED ISSUE #1366: Content for Error fetching event
**Describe the feature you'd like**
Show content when fetching event failed.

**Screenshots**

Screenshot_20190321-025723

**Would you like to work on the issue?**
- Yes

### PR DESCRIPTION
**Pull request by aman210697**
Fixes #1366 
Changes: new layout file for error on fetching event 

Screenshots for the change:
WhatsApp Image 2019-03-22 at 12 47 04 PM

### CODE REVIEW

**Comment by liveHarshit** [comment]
Resolve conflicts

**Comment by aman210697** [comment]
> Resolve conflicts

done.

File: app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json [hunk 1]
```diff
@@ -1074,4 +1074,4 @@
       "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, \"0b3cd25764884626e03f56b0600d1c76\")"
     ]
   }
-}
\ No newline at end of file
+}
```
Comment (liveHarshit) [inline app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json]: revert these changes

Comment (aman210697) [reply app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json]: After rebase, some lines of this file had changed. I was unable to revert this line
 https://drive.google.com/file/d/1Tesk-jCLuTA3ndC4gFc51yQumDkK3ELS/view?usp=sharing

Comment (aman210697) [reply app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json]: done.

**Comment by aman210697** [comment]
@iamareebjamal @liveHarshit what's wrong with Semantic?

**Comment by ShridharGoel** [comment]
@aman210697 Try adding a space after the colon in title

**Comment by aman210697** [comment]
@liveHarshit please review

**Comment by liveHarshit** [comment]
Remove snackbar message

**Comment by aman210697** [comment]
> Remove snackbar message

WhatsApp Image 2019-03-25 at 12 24 29 AM

**Comment by liveHarshit** [comment]
What happens when you click favorite or share button?

**Comment by aman210697** [comment]
> What happens when you click favorite or share button?

crash 😄

**Comment by liveHarshit** [comment]
> > What happens when you click favorite or share button?
> 
> crash

Then solve the issue properly.

**Comment by aman210697** [comment]
> > > What happens when you click favorite or share button?
> > 
> > 
> > crash
> 
> Then solve the issue properly.

WhatsApp Image 2019-03-25 at 11 38 45 PM

looks good?
